cloud_experts_tutorials/cloud-experts-using-cloudfront-and-waf.adoc

```diff
@@ -0,0 +1,359 @@
+:_content-type: ASSEMBLY
+[id="cloud-experts-using-cloudfront-and-waf"]
+= Tutorial: Using AWS WAF and Amazon CloudFront to protect ROSA workloads
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-using-cloudfront-and-waf
+
+toc::[]
+
+// Mobb content metadata
+// Brought into ROSA product docs 2023-09-21
+// ---
+// date: '2021-06-17'
+// title: Using CloudFront + WAF
+// aliases: ['/docs/aws/waf/cloud-front.md']
+// tags: ["AWS", "ROSA"]
+// authors:
+//  - 'Connor Wooley'
+// ---
+
+AWS WAF is a web application firewall that lets you monitor the HTTP and HTTPS requests that are forwarded to your protected web application resources.
+
+You can use an Amazon CloudFront to add a Web Application Firewall (WAF) to your {product-title} (ROSA) workloads. Using an external solution protects ROSA resources from experiencing denial of service due to handling the WAF.
+
+[id="prerequisites_{context}"]
+== Prerequisites
+
+* xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc#rosa-sts-creating-a-cluster-quickly[A ROSA Classic cluster].
+* You have access to the OpenShift CLI (`oc`).
+* You have access to the AWS CLI (`aws`).
+
+[id="environment-setup_{context}"]
+=== Environment setup
+
+* Prepare the environment variables:
++
+[source,terminal]
+----
+$ export AWS_PAGER=""
+$ export CLUSTER_NAME=$(oc get infrastructure cluster -o=jsonpath="{.status.infrastructureName}"  | sed 's/-[a-z0-9]\{5\}$//')
+$ export REGION=$(oc get infrastructure cluster -o=jsonpath="{.status.platformStatus.aws.region}")
+$ export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+$ export SCRATCH="/tmp/${CLUSTER_NAME}/cloudfront-waf"
+$ mkdir -p ${SCRATCH}
+$ echo "Cluster: ${CLUSTER_NAME}, Region: ${REGION}, AWS Account ID: ${AWS_ACCOUNT_ID}"
+----
+
+[id="custom_domain_setup{context}"]
+== Custom domain setup
+
+It is necessary to configure a secondary ingress controller to segment your external WAF-protected traffic from your standard (and default) cluster ingress controller. In ROSA, we do this using the Custom Domain Operator.
```

Comment:
Suggestion: 
s [throughout]/ingress controller/Ingress Controller

This is to match (most of the) Kubernetes references; the general recommendation from the networking team is to capitalize it.

Suggestion:
Remove the parentheses here:
"...standard and default cluster..."